ISSUE.md

```diff
@@ -0,0 +1,56 @@
+# Issue reporting on Podman
+
+The Podman team cares for our users and communities.  When someone has a problem with
+Podman and takes the time to report an [issue](https://github.com/containers/podman/issues)
+on our Github, we deeply appreciate the effort.  We want to help. Consider reading
+our [support](SUPPORT.md) document prior to submitting an issue as well.
+
+## Considerations when reporting an issue upstream
+
+### Where to report your issue
+
+If you are running Podman acquired from a Linux distribution and that Linux distribution has a
+bug reporting mechanism, then please report the bug there.
+
+### Development or latest version
+
+We view this Github repository as an upstream repository for development
+and the latest Podman version.
+
+The term "latest version" refers to our mainline development tree or the
+[latest release](https://github.com/containers/podman/releases/latest).
+
+You are welcome to replicate the bug with upstream code and report that.
+
+### Bugs vs features
+
+A bug is when something in Podman is not working as it should or has been described.  A
+feature or enhancement is when you would like Podman to behave differently.
+
+![new issue](issue-feature.png)
+
+### Use the issue template when reporting bugs
+When you report an issue on the upstream repository, be sure to fill out the entire template.
+You must provide the required `podman info` wherever possible as it helps us diagnose
+your report.  If possible, always provide a _reliable reproducer_.  This is extremely
+helpful for us during triage and bug fixing.
+
+### Look through existing issues before reporting a new issue
+Managing issues is a time consuming processes for maintainers.  You can save us time by
+making sure the issue you want to report has not already been reported.  It is appropriate
+to comment on the existing issue with relevant information.
+
+
+### Why was my issue report closed
+
+Issues filed upstream may be closed by a maintainer for the following reasons:
+
+* A fix for the issue has been merged into the main branch of our upstream
+repository.  It is possible that the bug was already fixed upstream as well.
+* The reported issue is a duplicate.
+* The issue is reported against a [distribution that has a bug reporting mechanism](#where-to-report-your-issue)
+or paid support.
+* The issue was reported using an [older version](#development-or-latest-version) of Podman.
+* A maintainer determines the code is working as designed.
+* The issue has become [stale](ISSUE-EOL-POLICY.md) and reporters are not responding.
+* We were unable to reproduce the issue, or there was insufficient information to reproduce the issue.
```

Comment:
Another one to add: The original reporter of the issue becomes unresponsive to questions for an extended period of time.

Reply:
i think this is covered in the line above?